### DISCUSSION TITLE
Home Assistant Energy Dash

### DISCUSSION THREAD

**Discussion by charlie-chester** [BottlecapDave/HomeAssistant-OctopusEnergy#800] [General]
Hi. I've set up the energy dash using the previous consumption rates. I find these more accurate. I've managed to set it up so the usage is shown on the correct date but when I add the entity to show the previous price it show that on today rather than yesterday. So the price is always one day out. 

Is this fixable?

**Comment by BottlecapDave** [comment]
Sorry for the late response.

As per the [docs](https://bottlecapdave.github.io/HomeAssistant-OctopusEnergy/setup/energy_dashboard/#for-electricity_1), you need to use the total accumulative cost sensor to get the correct cost associated with the data

**Comment by charlie-chester** [comment]
HiI’ve done all that and the usage shows on the correct date but the cost is always 1 day out. RegardsMark ChesterOn 7 Apr 2024, at 06:48, David Kendall ***@***.***> wrote:﻿
Sorry for the late response.
As per the docs, you need to use the total accumulative cost sensor to get the correct cost associated with the data

**Comment by BottlecapDave** [comment]
Hello, I think there was a bug in the docs regarding the cost sensor, where it should have been using the generated stats sensors instead of the entities (so that the costs are lined up). If you follow the updated docs, this should rectify your issue.
